client/src/components/BoxStyle.css

```diff
@@ -0,0 +1,48 @@
+.box-title {
```

Comment:
Please run Prettier (https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) on this file!

Reply:
Done 👍🏽